app/static/js/admin/event/wizard/vue-app.js

```diff
@@ -155,6 +155,9 @@ var app = new Vue({
         },
         'step': function (step) {
             history.replaceState(null, '', "./" + step);
+        },
+        'event.timezone': function (value) {
+            this.event.timezone = value;
```

Comment:
The event timezone value has to be updated to the current value in the wizard for the ticket even if the data has not been saved. Cannot use time when adding new ticket according to the filled timezone without this.